src/Dockerfile

```diff
@@ -3,6 +3,13 @@ FROM python:3.7.7-alpine3.12
 COPY . /popper
 
 RUN apk --no-cache add git && \
+    apk --no-cache add gcc && \
+    apk --no-cache add make && \
+    apk --no-cache add linux-headers && \
+    apk --no-cache add musl-dev && \
+    apk --no-cache add python3-dev && \
+    apk --no-cache add libffi-dev && \
+    apk --no-cache add openssl-dev && \
```

Comment:
this will make the image size significantly bigger. So one alternative is to remove all the build-time dependencies, but that might complicate things too much. The other option is to install the pre-packaged version of paramiko available in the edge branch of alpine, so something like this should work:

```dockerfile
FROM alpine:20200917

COPY . /popper

RUN apk --no-cache add git py3-pip py3-paramiko
    pip install --no-cache-dir /popper && \
    rm -r /popper
```

Reply:
Hi @ivotron! We will definitely look further into running with a pre-packaged version of paramiko. I'm not certain how to remove build-time dependencies from a pre-existing library, but I can research that as well if the pre-packaged version doesn't work out. 

Regarding the docstring changes, we will find and remove those as well!